sklearn/covariance/shrunk_covariance_.py

```diff
@@ -365,6 +365,25 @@ class LedoitWolf(EmpiricalCovariance):
         Coefficient in the convex combination used for the computation
         of the shrunk estimate.
 
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from sklearn.covariance import LedoitWolf
+    >>> n_samples, n_features = 2, 4
+    >>> np.random.seed(0)
+    >>> X = np.random.normal(size=(n_samples, n_features))
+    >>> lw = LedoitWolf()
+    >>> lw.fit(X)
```

Comment:
This kinda implies the output of the `fit` function is `None`, whereas it's `LedoitWolf(assume_centered=False, block_size=1000, store_precision=True)`. If you don't want to have the representation of the model printed, you can merge the two lines and have `lw = LedoitWolf().fit(X)`.

Reply:
I think the consensus is to add the output of the fit method, so @andreanr you can fix the tests by adding:

```
>>> lw.fit(X)
LedoitWolf(assume_centered=False, block_size=1000, store_precision=True)
```

Reply:
Yes, i'll fix it